xbmc/playlists/SmartPlayList.cpp

```diff
@@ -798,6 +805,8 @@ std::string CSmartPlaylistRule::FormatWhereClause(const std::string &negate, con
       query = GetField(m_field, strType) + " is NULL or " + GetField(m_field, strType) + parameter;
     else if (m_field == FieldSource)
       query = negate + " EXISTS (SELECT 1 FROM album_source, source WHERE album_source.idAlbum = " + table + ".idAlbum AND album_source.idSource = source.idSource AND source.strName" + parameter + ")";
+    else if (m_field == FieldDiscTitle)
+      query = negate + " songview.strDiscSubtitle" + parameter;
```

Comment:
Remove as strDiscSubtitle is a field of the songview,  hence SQL for rules about it is handled by adding to
https://github.com/xbmc/xbmc/blob/master/xbmc/utils/DatabaseUtils.cpp#L83
`else if (field == FieldDiscTitle) return "songview.strDiscSubtitle";`